contentreview-impl/impl/src/java/org/sakaiproject/contentreview/impl/hbm/BaseReviewServiceImpl.java

```diff
@@ -110,16 +110,134 @@ public void queueContent(String userId, String siteId, String taskId, String con
 		ContentReviewItem item = new ContentReviewItem(userId, siteId, taskId, contentId, new Date(),
 				ContentReviewItem.NOT_SUBMITTED_CODE);
 		item.setNextRetryTime(new Date());
+		item.setUrlAccessed(false);
+		dao.save(item);
+	}
+	
+	public void queueContent(String userId, String siteId, String taskId, String contentId, String submissionId)
+		throws QueueException {
+	
+		log.debug("Method called queueContent(" + userId + "," + siteId + "," + contentId + ")");
+
+		if (userId == null) {
+			log.debug("Using current user");
+			userId = userDirectoryService.getCurrentUser().getId();
+		}
+
+		if (siteId == null) {
+			log.debug("Using current site");
+			siteId = toolManager.getCurrentPlacement().getContext();
```

Comment:
Does this really need to be to be here as having a service depend on context setup by the portal can lead to unexpected behaviour.